### DISCUSSION TITLE
Customer based prices

### DISCUSSION THREAD

**Discussion by danielpervan** [saleor/saleor#7122] [Q&A]
We're evaluating a few different e-commerce platforms, but one critical requirement is be able to also handle certain B2B customers with custom price lists. Would that be possible to do with Saleor? Either as core functionality, plugins or even just a suitable workaround.

We're probably only going to use the backend and admin dashboard, not the storefront.

Otherwise, has there been any talks about adding price lists in the future? I can't seem to find anything about that on the roadmap.

**Comment by krzysztofwolski** [comment]
You could create separate channels for some groups of b2b clients (more info about channels: https://docs.saleor.io/docs/3.0/dashboard/channels). Not sure if I fully get your use case, but I would configure:
- default-channel: default prices for any customer
- b2b-channel: different prices tier for b2b clients

Please note, there's no additional backend check if certain users should have access to the channel - all the client needs to know is the channel slug.
